benchmarks/user/README.md

```diff
@@ -211,6 +211,51 @@ Do not forget to pin processes, especially on NUMA systems; below, we do so with
 ```
 numactl --cpubind=0 --membind=0 python benchmark.py ...
 ```
+
+## Benchmarking performance
+
+In order for a user to benchmark the performance of an operator calculating
+GFlops/s, Operational Intensity and Execution timings the `bench` mode should
+be used while also providing as an environment variable `DEVITO_LOGGING=DEBUG`.
+
+So the isotropic acoustic wave forward Operator in a `512**3` grid, space order
+12, and a simulation time of 100ms:
+
+```
+`DEVITO_LOGGING=DEBUG` python benchmark.py bench -P acoustic -d 512 512 512 -so 12 --tn 100
+```
+
+A `.json` file will be created in a folder named `results` except if otherwise specified
+with the `-r` option specifying the results directory.
+
+
+## Plotting a roofline
```

Comment:
Generation of roofline models ?

Reply:
is "plotting" the right word? it's more than a "simple chart/plot" at the end of the day. Maybe I'm being too picky...